benchmark/test_correctness_medical_usecase.py

```diff
@@ -0,0 +1,110 @@
+import re
+import inspect
+
+import nltk
+import pytest
+
+from biochatter._misc import ensure_iterable
+from .conftest import calculate_bool_vector_score
+from .benchmark_utils import (
+    categorize_failure_modes,
+    skip_if_already_run,
+    get_result_file_path,
+    write_results_to_file,
+    get_failure_mode_file_path,
+    write_failure_modes_to_file,
+)
+
+
+def test_correctness_of_answers(
+    model_name,
+    test_data_correctness,
+    conversation,
+    multiple_testing,
+):
+    # Downloads the naturale language synonym toolkit, just need to be done once per device
+    # nltk.download()
```

Comment:
This is not a good solution. Often, we execute runs on remote hardware or GitHub runners, where we can't "manually uncomment and install something."

I suggest you find a way to programmatically do this. IIRC, NLTK only downloads if it's not in cache. If not, I would probably resort to a Pytest fixture that checks whether it is there, and only downloads if it is not.